src/scss/config/_scale.scss

```diff
@@ -29,7 +29,6 @@ $font-sizes: (
   'small': 0.8rem,
   'xsmall': 0.7rem,
   'xxsmall': 0.6rem,
-  'rem': calc(1.25em + 0.5vw),
```

Comment:
I guess another option here, if we think it's valuable to display the root size, would be to replace this with something like `'root': 1rem`?